docs/content/contribute/review-pr.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Review a PR"
+weight: 11
+---
+
+# Review a PR
```

Comment:
There are different expectations for maintainer and contributor PRs (self-service / community), should we call them out separately in some way? In particular I'm thinking of:

- I don't give more than a passing glance to release notes for a maintainer PR, and trust they've written them correctly. For contributor PRs I've got to author them myself.
- For contributors I've got to wait until all tests are green until clicking approve. For maintainers, I'm generally fine approving if I'm satisfied with the code as-written and tests are most of the way (80-90%) there, even if they're currently failing.

Reply:
FWIW I would prefer not to call out that we might have separate standards for maintainers since that is not relevant to most of the people who would read this page.

Reply:
Ehh, fair enough- the page is doing double-duty as a maintainer and contributor guide in that case, which may leave ambiguities for maintainers.